scripts/ci-frontend-metrics.sh

```diff
@@ -27,4 +39,9 @@ echo "Metrics: {
   \"grafana.ci-code.grafana-ui.mdx\": \"${MDX_COUNT}\",
   \"grafana.ci-code.legacyForms\": \"${LEGACY_FORMS}\",
   \"grafana.ci-code.strictLint.noExplicitAny\": \"${STRICT_LINT_EXPLICIT_ANY}\"
+  \"grafana.ci-code.bundleFolderSize\": \"${TOTAL_BUNDLE}\",
+  \"grafana.ci-code.dependencies.outdated\": \"${OUTDATED_DEPENDENCIES}\",
+  \"grafana.ci-code.dependencies.lowVulnerabilities\": \"${LOW_VULNERABILITIES}\",
+  \"grafana.ci-code.dependencies.mediumVulnerabilities\": \"${MED_VULNERABILITIES}\",
+  \"grafana.ci-code.dependencies.highVulnerabilities\": \"${HIGH_VULNERABILITIES}\",
```

Comment:
I guess we could also have the following structure:
```suggestion
  \"grafana.ci-code.dependencies.vulnerabilities.low\": \"${LOW_VULNERABILITIES}\",
  \"grafana.ci-code.dependencies.vulnerabilities.medium\": \"${MED_VULNERABILITIES}\",
  \"grafana.ci-code.dependencies.vulnerabilities.high\": \"${HIGH_VULNERABILITIES}\",
```

In this case we can more easily count the total for vulnerabilities, WDYT?

Reply:
Excellent suggestion Hugo, thanks! 🥳